lua/quick_buffer_jump.lua

```diff
@@ -1,5 +1,13 @@
 local M = {}
 
+M.config = {
+    alphabet = "abcdefghijklmnopqrstuvwxyz",
```

Comment:
Thanks for the PR!

Please note `jkq` are reserved alphabets. They are essential for internal functionality of the plugin.

These alphabets are safe to use `abcdefghilmnoprstuvwxyz`

Reply:
I did not realize. I have updated the default alphabet option to be correct and included code to check for the existence of the reserved characters and remove them if present!